frontend/src/app/state/exercise/exercise.selectors.ts

```diff
@@ -29,6 +31,13 @@ export const getSelectMapImage = (mapImageId: UUID) => (state: AppState) =>
     state.exercise.mapImages[mapImageId];
 export const getSelectVehicle = (vehicleId: UUID) => (state: AppState) =>
     state.exercise.vehicles[vehicleId];
+export const getSelectVehicleTemplate =
+    (vehicleTemplateId: UUID) => (state: AppState) =>
+        state.exercise.vehicleTemplates.find(
+            (vehicleTemplate) => vehicleTemplate.id === vehicleTemplateId
+        )!;
```

Comment:
Is this `!` a safe assumption?

Reply:
It is as safe an assumption as with the other such selectors. See #358

Maybe put a todo comment there - I would try to tackle these problems in the codebase all together.